src/common/src/types/struct_type.rs

```diff
@@ -45,7 +45,7 @@ impl Debug for StructType {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Clone, Debug, PartialOrd, Ord)]
```

Comment:
We may also override the derived impl of `Ord`. The behavior should be consistent with `Eq`.